apisix/plugins/key-auth.lua

```diff
@@ -110,6 +114,10 @@ function _M.rewrite(conf, ctx)
     end
     core.log.info("consumer: ", core.json.delay_encode(consumer))
 
+    if conf.hide_credentials then
```

Comment:
We should remove the key according to where the key comes from.

Reply:
ok, I will.